addon/components/docs-viewer/x-main/index.hbs

```diff
@@ -41,7 +42,8 @@
       <LinkTo
         class='docs-text-grey-darkest docs-text-large-4 docs-font-light docs-no-underline
         docs-border-b docs-border-grey hover:docs-border-grey-darkest docs-transition'
-        @params={{this.docsRoutes.next.route}}
+        @route={{this.docsRoutes.next.route}}
```

Comment:
Same with these

Reply:
@rwwagner90 yes, this works in combination with `@models` argument